.travis.yml

```diff
@@ -11,36 +11,47 @@ install:
   - 'if [ -n "${PACKAGE-}" ]; then cd "packages/${PACKAGE}"; fi'
   - 'if [ "${TRAVIS_NODE_VERSION}" = "0.6" ] || [ "${TRAVIS_NODE_VERSION}" = "0.9" ]; then nvm install --latest-npm 0.8 && npm install && nvm use "${TRAVIS_NODE_VERSION}"; else npm install; fi;'
   - 'if [ -n "${ESLINT}" ]; then npm install --no-save "eslint@${ESLINT}"; fi'
+  - 'if [ -n "${REACT_HOOKS}" ]; then npm install --no-save "eslint-plugin-react-hooks@${REACT_HOOKS}"; fi'
 script:
   - 'if [ -n "${PREPUBLISH-}" ]; then npm run pretravis && npm run prepublish && npm run posttravis; elif [ -n "${LINT-}" ]; then npm run lint; else npm run travis; fi'
 sudo: false
 env:
   matrix:
     - 'TEST=true ESLINT=6 PACKAGE=eslint-config-airbnb-base'
     - 'TEST=true ESLINT=6 PACKAGE=eslint-config-airbnb'
+    - 'TEST=true ESLINT=6 REACT_HOOKS=1.7 PACKAGE=eslint-config-airbnb'
     - 'TEST=true ESLINT=5 PACKAGE=eslint-config-airbnb-base'
     - 'TEST=true ESLINT=5 PACKAGE=eslint-config-airbnb'
+    - 'TEST=true ESLINT=5 REACT_HOOKS=1.7 PACKAGE=eslint-config-airbnb'
 matrix:
   fast_finish: true
   include:
     - node_js: "lts/*"
       env: PREPUBLISH=true ESLINT=6 PACKAGE=eslint-config-airbnb-base
     - node_js: "lts/*"
       env: PREPUBLISH=true ESLINT=6 PACKAGE=eslint-config-airbnb
+    - node_js: "lts/*"
+      env: PREPUBLISH=true ESLINT=6 REACT_HOOKS=1.7 PACKAGE=eslint-config-airbnb
     - node_js: "lts/*"
       env: PREPUBLISH=true ESLINT=5 PACKAGE=eslint-config-airbnb-base
     - node_js: "lts/*"
       env: PREPUBLISH=true ESLINT=5 PACKAGE=eslint-config-airbnb
+    - node_js: "lts/*"
+      env: PREPUBLISH=true ESLINT=5 REACT_HOOKS=1.7 PACKAGE=eslint-config-airbnb
     - node_js: "lts/*"
       env: LINT=true
     - node_js: "7"
       env: TEST=true ESLINT=5 PACKAGE=eslint-config-airbnb-base
     - node_js: "7"
       env: TEST=true ESLINT=5 PACKAGE=eslint-config-airbnb
+    - node_js: "7"
+      env: TEST=true ESLINT=5 REACT_HOOKS=1.7 PACKAGE=eslint-config-airbnb
     - node_js: "6"
       env: TEST=true ESLINT=5 PACKAGE=eslint-config-airbnb-base
     - node_js: "6"
       env: TEST=true ESLINT=5 PACKAGE=eslint-config-airbnb
+    - node_js: "6"
+      env: TEST=true ESLINT=5 REACT_HOOKS=1.7 PACKAGE=eslint-config-airbnb
   exclude:
   allow_failures:
```

Comment:
I think the two new `PREPUBLISH` entries under `include` need equivalent entries under `allow_failures` here (since currently those runs are failing). eg:

```yaml
    - env: PREPUBLISH=true ESLINT=6 REACT_HOOKS=1.7 PACKAGE=eslint-config-airbnb
    - env: PREPUBLISH=true ESLINT=5 REACT_HOOKS=1.7 PACKAGE=eslint-config-airbnb
```

Reply:
They don't; they're failing by design on master since new eslint rules have been released that haven't yet been added to the config. I'll rebase your PR after I've fixed it, you don't have to worry about it.